### PR TITLE
84599: Customer Insights Added to KOT UI Tutorial

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/84599/add-insights-to-kots-tutorial

Preview: https://deploy-preview-1346--replicated-docs-upgrade.netlify.app/vendor/tutorial-ui-deploy-app

### CODE REVIEW

File: docs/vendor/tutorial-ui-deploy-app.md [hunk 1]
```diff
@@ -80,6 +80,12 @@ To install and deploy the application:
 
   If you are using a VM that is behind a firewall and you get an error message, you might need to create a firewall rule to enable access to port 8800. For more information, see [Installation Requirements](/enterprise/installing-general-requirements).
 
+1. Under **Customers** in the vendor portal, select the customer name to open the **Reporting** page. In the **Instances** pane, you can verify that the instance is active and drill down from the details area to see the instance details. For more information, see [Customer Reporting](customer-reporting) and [Instance Details](instance-insights-details).
```
Comment (jonquil2002) [inline docs/vendor/tutorial-ui-deploy-app.md:83]: @paigecalvert Do you think this is enough information? I was hesitant to create a new topic in the process to show off this feature.

Comment (paigecalvert) [reply docs/vendor/tutorial-ui-deploy-app.md:83]: Yeah I think that's perfect! Letting them know it's there feels sufficient.

Comment (paigecalvert) [reply docs/vendor/tutorial-ui-deploy-app.md:83]: One note to consider: Could add one more mention of it after the user upgrades their app from the admin console to let them know that they can go back to the vendor portal to see the update event reflected in the instance data for the customer (I think the quick start has an example of this)

**Review by paigecalvert — DISMISSED** [review]
looks great! one suggestion for your consideration. could be a follow up PR

File: docs/vendor/tutorial-ui-deploy-app.md (hunk 1 above)
Comment (jonquil2002) [reply docs/vendor/tutorial-ui-deploy-app.md]: Good idea. Done.

**Pull request merged**